packages/logs/README.md

```diff
@@ -667,6 +667,33 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
 
 **Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the SDK.
 
+### Access internal context
+
+After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This will allow you to access the `session_id`.
+
+```
+getInternalContext (startTime?: 'number' | undefined)
+```
+You can optionally use `startTime` parameter to get the context of a specific time. If not return the current context is parameter is omitted.
+
+#### CDN async
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏to be consistent with other sections, we should also have a paragraph for npm setup